Tests/UIKitTests/UICollectionViewExtensionsTests.swift

```diff
@@ -53,6 +53,13 @@ final class UICollectionViewExtensionsTests: XCTestCase {
 			XCTAssert(completionCalled)
 		}
 	}
+    
+    func testRegisterCellWithNibUsingClass() {
```

Comment:
Now it errors. Maybe the resources aren't available to the test target? Can't look into it further

Reply:
My guess is that due to the removal of the `#if os(iOS)`, it executes the test on tvOS. Correct me if I'm wrong. Also, I've seen the same pattern to some tests in the UITableViewExtensionsTest that's why I implemented the same on my tests

Reply:
Will revert back to previous implementation since my tests consists of using an .xib file which is not supported for tvOS

Reply:
If the method is available on `tvOS`, as declared in the source file, we shouldn't be restricting the test to `iOS` only. I'm wondering if the nib resources are not part of the `tvOS` target, causing the test to fail

Reply:
I see. I managed to make the tests succeed now. Will update PR

Reply:
So it looks like it still fails. Maybe we should only make the source available to iOS? We should not expose them to tvOS if we can't get it to work.

Reply:
I believe so. So I'll just update the UITableView and UICollectionView extensions that make use of nibs